requirements.txt

```diff
@@ -1,5 +1,5 @@
-PyJWT==1.1.0
-slumber==0.7.1
+# Package requirements
+-e .
```

Comment:
Thinking of local development, we should keep the `-e` so that developers can easily make changes without having to uninstall/reinstall this package. Also, @dan-f is correct in stating that `requirements.txt` will never be used for production usage. Anyone who is doing that deserves what he/she gets.